renderers/src/test/java/com/pedrogomez/renderers/IntegerRenderer.java

```diff
@@ -0,0 +1,35 @@
+package com.pedrogomez.renderers;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+/**
+ * Renderer created only for testing purposes.
+ *
+ * @author Rostyslav Roshak.
```

Comment:
Can you add the project's copyright header and remove the author? In this project we are not using ``@author`` annotations :)